components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -278,6 +275,16 @@ - (NSString *)accessibilityLabelWithTitle:(NSString *)title {
   return [labelComponents componentsJoinedByString:@", "];
 }
 
+- (CGPoint)badgeCenterFrom:(CGRect)iconFrame rtl:(BOOL)rtl {
```

Comment:
Nit/naming: Please include the parameter names in the method signature, BOOLs should usually have an `is` prefix, prefer "for argument" instead of "from argument".

```
- (CGPoint)badgeCenterForIconFrame:(CGRect)iconFrame isRTL:(BOOL)isRTL {
```

Reply:
Please include the "IconFrame" parameter name in the method signature.